### PR TITLE
Login with LDAP

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by johngian** [comment]
Please use flake8 to identify and fix code styling issues.

File: pages/home_page.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            self.selenium.get(self.base_url)
+
+    def click_sign_in_button(self):
+        self.selenium.find_element(*self._sign_in_button).click()
+
+    def ldap_login(self, email_address, password):
+        self.click_sign_in_button()
+        auth0 = Auth0(self.base_url, self.selenium)
+        auth0.login_with_ldap(email_address, password)
+        from pages.two_factor_authentication_page import TwoFactorAuthenticationPage
```
Comment (johngian) [inline pages/home_page.py]: Please move this to import section.

File: pages/two_factor_authentication_page.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    def enter_passcode(self, passcode):
+        self.selenium.switch_to_frame('duo_iframe')
+        self.wait_for_element_visible(*self._enter_passcode_button)
+        self.selenium.find_element(*self._enter_passcode_button).click()
+        self.selenium.find_element(*self._passcode_field_locator).send_keys(passcode)
+        self.selenium.find_element(*self._enter_passcode_button).click()
+        WebDriverWait(self.selenium, self.timeout).until(
+            lambda s: s.find_element(*self._successfull_passcode_message_locator).text == "Success! Logging you in...")
+        self.selenium.switch_to_default_content()
+        from authentication_status_page import AuthenticationStatusPage
```
Comment (johngian) [inline pages/two_factor_authentication_page.py]: Same here.

File: requirements/flake8.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+flake8==3.2.1
```
Comment (johngian) [inline requirements/flake8.txt]: Please rename this file to ``dev.txt`` to make it more generic and allow adding further requirements in the future.

**Comment by johngian** [comment]
r+wc

File: requirements/flake8.txt (hunk 3 above)
Comment (viorelaioia) [inline requirements/flake8.txt]: renamed

**Pull request merged**